packages/ember-htmlbars/lib/helpers/if_unless.js

```diff
@@ -64,6 +64,7 @@ function appendBlockConditional(view, inverted, helperName, params, hash, option
     view.appendChild(BoundIfView, {
       _morph: options.morph,
       _context: get(view, 'context'),
+      renderer: get(view, 'renderer'),
```

Comment:
Can this be `view.renderer`? And why is the `renderer` not set in the other views, e.g. in the with helper?